tooling/cli/Cargo.toml

```diff
@@ -52,7 +52,7 @@ toml_edit = "0.14"
 json-patch = "0.2"
 tauri-utils = { version = "1.1.1", path = "../../core/tauri-utils", features = [ "isolation", "schema", "config-json5", "config-toml" ] }
 toml = "0.5"
-valico = "3.6"
```

Comment:
`valico` seems to be unmaintained and it has a bug with its pattern parser see https://github.com/s-panferov/valico/issues/73